lib/oneview-sdk/resource/api200/logical_interconnect_group.rb

```diff
@@ -53,14 +47,25 @@ def self.get_default_settings(client)
       # @param [String] type Interconnect type
       # @raise [StandardError] if a invalid type is given then raises an error
```

Comment:
@ raise [StandardError] if ~a~ an invalid type is given ~then raises an error~
Remember YARD will parse it to something like: **raises StandardError if an invalid type is given**